usolitaire/game.py

```diff
@@ -252,3 +252,127 @@ def can_move_to_foundation_from_tableau(self, index) -> bool:
     def won(self):
         """Check if the game is won"""
         return all(len(pile) == 13 for pile in self.foundations)
+
+
+class FreecellGame(Game):
```

Comment:
Okay, can you now implement the user interface for this in a new src/freecell_app.py file? You can get inspired from the code in src/app.py which implements it for Klondike.